packages/app/cypress/e2e/specs_list_e2e.cy.ts

```diff
@@ -3,6 +3,21 @@ describe('App: Spec List (E2E)', () => {
     cy.scaffoldProject('cypress-in-cypress')
     cy.openProject('cypress-in-cypress')
     cy.startAppServer('e2e')
+
+    cy.withCtx((ctx) => {
+      const yesterday = new Date()
+
+      yesterday.setDate(yesterday.getDate() - 1)
+
+      sinon.stub(ctx.git, 'gitInfo').callsFake(() => {
+        return Promise.resolve({
+          author: 'Test Author',
+          lastModifiedTimestamp: yesterday.toDateString(),
+          lastModifiedHumanReadable: yesterday.toDateString(),
+        })
+      })
+    })
```

Comment:
Since we display relative dates in the UI "Modified X days ago by ...", I'm stubbing this out as "yesterday" so that we get consistent rendered output.